cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/CPKDependenciesTask.kt

```diff
@@ -109,127 +77,79 @@ open class CPKDependenciesTask @Inject constructor(objects: ObjectFactory) : Def
         val digest = digestFor(hashAlgorithm.get().toUpperCase())
 
         try {
-            val xmlDocument = createXmlDocument()
-            val writer = DependencyWriter(xmlDocument, digest)
-
-            projectCpks.forEach { cpk ->
-                logger.info("Project CorDapp CPK dependency: {}", cpk.name)
-                JarFile(cpk).use(writer::writeProjectDependency)
-            }
-
-            remoteCpks.forEach { cpk ->
-                logger.info("Remote CorDapp CPK dependency: {}", cpk.name)
-                JarFile(cpk).use(writer::writeRemoteDependency)
+            // Write CPK dependency information as JSON document.
+            cpkOutput.get().asFile.printWriter(Charset.forName("UTF-8")).use {
```

Comment:
I was going to request that you use `java.nio.charset.StandardCharsets.UTF_8` instead of `Charset.forName("UTF-8")` here, but actually I think [UTF-8 is the default anyway](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/print-writer.html).

Reply:
removed charset